dts/riscv/nordic/nrf54h20_cpuflpr.dtsi

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <nordic/nrf54h20.dtsi>
+
+cpu: &cpuflpr {};
+clic: &cpuflpr_clic {};
+cpuppr_vevif: &cpusys_vevif_tx {};
+cpuflpr_vevif: &cpuflpr_vevif_rx {};
+cpusys_vevif: &cpusys_vevif_tx {};
+
+/delete-node/ &cpuapp;
+/delete-node/ &cpuapp_peripherals;
+/delete-node/ &cpuapp_ppb;
+/delete-node/ &cpuapp_ram0;
+/delete-node/ &cpurad;
+/delete-node/ &cpurad_peripherals;
+/delete-node/ &cpurad_ppb;
+/delete-node/ &cpurad_ram0;
+/delete-node/ &cpuppr;
+
+/ {
+	soc {
+		compatible = "simple-bus";
+		interrupt-parent = <&cpuflpr_clic>;
+		ranges;
+	};
+};
+
+&cpuflpr_clic {
+	status = "okay";
+};
+
+&cpusec_bellboard {
+	compatible = "nordic,nrf-bellboard-tx";
+};
+
+&cpuapp_bellboard {
+	compatible = "nordic,nrf-bellboard-tx";
+};
+
+&cpurad_bellboard {
+	compatible = "nordic,nrf-bellboard-tx";
+};
```

Comment:
This also should be removed. FLPR should not communicate with Radio. Only with App.

Reply:
done